waiter/src/waiter/token.clj

```diff
@@ -85,9 +85,9 @@
 
 (defn send-internal-index-event
   "Send an internal event to be processed by the tokens-watch-maintainer daemon process"
-  [tokens-update-chan token owner]
-  (log/info "sending internal index event" {:token token :owner owner})
-  (async/put! tokens-update-chan {:owner owner :token token}))
+  [tokens-update-chan token]
+  (log/info "sending internal index event" {:token token})
+  (async/put! tokens-update-chan token))
```

Comment:
Let's continue to forward a map to `tokens-update-chan`. We can add in more items later if we need to easily then.